### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by wilyle**
Takes all of the updates from https://github.com/eclipse-ibeji/ibeji/pull/95 except for `http`, `http-body`, `hyper`, and `sdl2` which have various incompatibilities with Ibeji. These updates have been lingering for months, and the incompatible versions of tonic between Ibeji and Chariott are causing build issues in Freyja

### CODE REVIEW

File: external/iot-plugandplay-models
Comment (ashbeitz) [inline external/iot-plugandplay-models:1]: Not sure why this needed to be updated.

File: external/opendigitaltwins-dtdl
Comment (ashbeitz) [inline external/opendigitaltwins-dtdl:1]: Not sure why this needed to be updated.

**Review by ashbeitz — APPROVED** [review]
Approved with two comments.

File: external/opendigitaltwins-dtdl
Comment (wilyle) [reply external/opendigitaltwins-dtdl:1]: see my other comment

**Review by ashbeitz — APPROVED** [review]
Approved

**Pull request merged**